gef-ui/server/controllers/amendments/cover-end-date/validation.test.ts

```diff
@@ -0,0 +1,92 @@
+import { applyStandardValidationAndParseDateInput, COVER_END_DATE_MAXIMUM_YEARS_IN_FUTURE, DayMonthYearInput } from '@ukef/dtfs2-common';
+import { add, format, startOfDay, getUnixTime } from 'date-fns';
+import { validateAndParseCoverEndDate } from './validation';
+import { mapValidationError } from '../../../utils/map-validation-error';
+
+const valueName = 'cover end date';
+const valueRef = 'coverEndDate';
+
+const getFormattedDate = (coverEndDate: Date): DayMonthYearInput => {
+  return {
+    day: format(coverEndDate, 'd'),
+    month: format(coverEndDate, 'M'),
+    year: format(coverEndDate, 'yyyy'),
+  };
+};
+
+describe('validateAndParseCoverEndDate', () => {
+  afterAll(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should map the response from applyStandardValidationAndParseDateInput', () => {
+    // Arrange
```

Comment:
Can you please add an assertion with all fields `day`, `month` and `year` all are falsly i.e. '' and null.